### DISCUSSION TITLE
NFC Tag - SKU Inventory Tracker

### DISCUSSION THREAD

**Discussion by kasiajeanne** [inventree/InvenTree#5757] [Ideas]
This is my first time starting a discussion here (I apologize if I'm not formatting or explaining correctly).

I'm a record label manager for an independent label & I have to produce & maintain the inventory for physical records & merch for our artists. We have an issue within the office since we're currently using stickers to keep track between (way too frequent) full counts.

I am trying to implement a tagging system that would link physical NFC stickers & whenever we grab a unit, we tap & it adjusts in the system.

Maybe someone here has an idea in how this could be done or point out a way of doing it that already exists?

**Comment by SchrodingersGat** [comment]
@kasiajeanne welcome to the community! That's an interesting requirement. We have an [idea to support NFC in the mobile app](https://github.com/inventree/inventree-app/issues/85) - if implemented, this might be what you are looking for?
